projects/igniteui-angular/src/lib/core/utils.ts

```diff
@@ -622,3 +622,26 @@ export function* intoChunks<T>(arr: T[], size: number) {
     yield arr.slice(i, i + size);
   }
 }
+
+/**
+ * @param size 
+ * @returns string that represents the --component-size default value 
+ */
+export function getComponentCssSizeVar(size: string) {
+    switch (size) {
+        case "1":
+            return 'var(--ig-size, var(--ig-size-small))';
+        case "2":
+            return 'var(--ig-size, var(--ig-size-medium))';
+        case "3":
+        default:
+            return 'var(--ig-size, var(--ig-size-large))';
+    }
+}
+
+/**
+ * Sets element size as a inline style
+ */
+export function setElementSize(element: HTMLElement, size: string) {
```

Comment:
If that's used by tests only, it could and should live in the test utils? See projects/igniteui-angular/src/lib/test-utils/